### PR TITLE
feat: add fuzer tracker

### PR DESCRIPTION
**Pull request by kobik**

### CODE REVIEW

File: definitions/v3/fuzer.yml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    details:
+      selector: a[href^="/showthread.php?t="]
+      attribute: href
+    download:
+      selector: a[href^="/attachment.php?attachmentid="]
+      attribute: href
+    size:
+      selector: td:nth-child(4)
+    grabs:
+      selector: td:nth-child(5)
```
Comment (kobik) [inline definitions/v3/fuzer.yml:139]: when there are 1000 or more grabs, i'm facing the following error
`Error while parsing field=grabs, selector=td:nth-child(5), value=1,683: Value was either too large or too small for an Int32.`
how should i solve it?

Comment (bakerboy448) [reply definitions/v3/fuzer.yml:139]: @Qstick do we need to change cardigann to use Long instead?

Comment (austinwbest) [reply definitions/v3/fuzer.yml:139]: It needs to not have a comma

Comment (Qstick) [reply definitions/v3/fuzer.yml:139]: Yes the parser doesn't like ints with commas, do a replace filter on that prop in the yml to replace `","` with `""`

Comment (kobik) [reply definitions/v3/fuzer.yml:139]: thanks.

replaced commas in relevant fields

**Comment by kobik** [comment]
@Qstick it's now ready for a review

**Comment by Qstick** [comment]
Lgtm

**Comment by kobik** [comment]
Can we make it @Qstick ?

Thanks

**Pull request merged**